config/locales/en/action.yml

```diff
@@ -0,0 +1,5 @@
+en:
```

Comment:
I looked at the locale file in the Simple Form repo and it doesn't have these strings